p2p/p2p.go

```diff
@@ -36,14 +36,11 @@ const SendStatusInterval = 5 * time.Minute
 
 // Service describes a p2p service
 type Service struct {
-	ctx          context.Context
-	host         *host
-	msgRec       <-chan Message
-	msgSend      chan<- Message
-	blockAnnRec  map[string]bool
-	blockReqRec  map[string]bool
-	blockRespRec map[string]bool
-	txMessageRec map[string]bool
+	ctx     context.Context
+	host    *host
+	gossip  *Gossip
```

Comment:
Should `Gossip` be `gossip` or `host` be `Host` or is this right?

Reply:
should be `gossip` as we don't plan to export it

Reply:
fixed